packages/sampleProp.conf

```diff
@@ -0,0 +1,23 @@
+[DEFAULT]
+GatewayName = Gateway-Name
+DeviceName = Gateway-Name
+Metric1Name = Metric1-Name
+Metric2Name = Metric2-Name
+Metric3Name = Metric3-Name
+Metric4Name = Metric4-Name
+
+GatewayPropList = {"Country":"USA-G", "State":"California", "City":"Palo Alto", "Location":"VMware HQ", "Building":"Promontory H Lab", "Floor":"First Floor"}
+
+[GRAPHITE]
+IP = Graphite-IP-Address
+Port = Graphite-Port
+
+[BIKE_SIMULATOR]
```

Comment:
My idea of implementing sections was to be easily able to club properties related to DCC, DCCComms, DeviceComms, DefaultPropertiesForAllPrograms etc. I don't think we should have different sections for each simulator. Why can't the Bike Simulator or Thermistor Simulator reuse the properties from the DEFAULT section?